src/main/java/org/elasticsearch/index/translog/fs/FsTranslog.java

```diff
@@ -435,144 +518,124 @@ public void syncOnEachOperation(boolean syncOnEachOperation) {
         }
     }
 
-    @Override
-    public Path getPath(long translogId) {
+    /** used for testing */
+    Path getPath(long translogId) {
```

Comment:
This should change as per https://github.com/elastic/elasticsearch/pull/10621 (might happen with a rebase/master-merge)